### PR TITLE
rust webpack template tutorial

### PR DESCRIPTION
**Pull request by DebugSteven**
This is a work in progress PR for the rust webpack template tutorial.

At the moment, I believe the workflow to use the template is filled in pretty well. There are empty WIP files to talk more about the generated `src/lib.rs` file & the `web-sys` dependency in the `Cargo.toml`, in addition to the other dependencies & crate type that are [discussed in the npm-browser-packages template tutorial](https://github.com/rustwasm/wasm-pack/blob/marketing/docs/src/tutorials/npm-browser-packages/template-deep-dive/cargo-toml.md).

I'd appreciate feedback on content so far & possibly what to do about duplication across tutorials, if that's a priority.

### CODE REVIEW

**Comment by danwilhelm** [comment]
If you'd like to save some time writing the deep dives, the `Cargo.toml` and `lib.rs` auto-generated by `npm init rust-webpack` are extremely similar to those described in the existing deep dives here: https://github.com/rustwasm/wasm-pack/tree/master/docs/src/tutorial/template-deep-dive

**Comment by DebugSteven** [comment]
Hey @xtuc this PR is ready for review. I'm happy to make changes based on any feedback. Thanks for offering to look this over.

File: docs/src/tutorials/hybrid-applications-with-webpack/using-your-library.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Now, open up the `js/index.js` file. We see our Rust-generated wasm `run` function being
+called inside our JS file.
+
+```js
+import("../crate/pkg").then(module => {
+  module.run();
+});
+```
+
+## Run The Project
```
Comment (xtuc) [inline docs/src/tutorials/hybrid-applications-with-webpack/using-your-library.md]: This isn't needed, Webpack's wasm-pack-plugin will spawn wasm-pack. Only `npm start` or `npm run build` in the root directory is needed.

Comment (DebugSteven) [reply docs/src/tutorials/hybrid-applications-with-webpack/using-your-library.md]: Thank you for explaining that! I pushed up a change that I think reflects that a little better in the docs now.

**Comment by ashleygwilliams** [comment]
@DebugSteven is this ready for review? Let me know! And thanks for all the effort! I'm excited for this to land :)

**Comment by COGoldRust** [comment]
@ashleygwilliams It's ready for review! Thanks for your patience.

**Comment by DebugSteven** [comment]
@ashleygwilliams It's ready for review! Thanks for your patience.

**Pull request merged**